docs/protocol.md

```diff
@@ -1,230 +0,0 @@
-# The GSN Protocol
```

Comment:
why remove, and not update?

Reply:
It will never be well-maintained. We have https://github.com/opengsn/docs.opengsn.org for documentation in version control